integration/tests/cook/test_basic.py

```diff
@@ -1422,6 +1436,10 @@ def test_400_on_group_query_without_uuid(self):
         resp = util.query_groups(self.cook_url)
         self.assertEqual(400, resp.status_code)
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
```

Comment:
Can we have this string in a constant somewhere and refer to the constant in the annotation?

Reply:
Done